fhir/fhir_connector.bal

```diff
@@ -71,7 +71,7 @@ public isolated client class FHIRConnector {
     # + summary - The subset of the resource content to be returned
     # + return - If successful, FhirResponse record else FhirError record
     @display {label: "Get resource by the logical ID"}
-    remote isolated function getById(@display {label: "Resource Type"} ResourceType|string 'type,
+    isolated resource function get read/id(@display {label: "Resource Type"} ResourceType|string 'type,
```

Comment:
If we don't need annotation to be attached to param `id` we can rewrite this method using path params as `isolated resource function get read/[string id](@display {label: ...` See https://ballerina.io/learn/by-example/http-path-param/

We can call this method like `check fhirConnector->/read/pat1(PATIENT);`

If `pat1` is a variable we can also do
```bal
string id = "pat1";
check fhirConnector->/read/[id](PATIENT);
```

Reply:
Lets update similar places as well